health/notifications/telegram/README.md

```diff
@@ -19,7 +19,7 @@ You need to:
 2.  Start a conversation with your bot or invite it into a group where you want it to send messages.
 3.  Find the chat ID for every chat you want to send messages to. Contact the [@myidbot](https://t.me/myidbot) bot and send the `/getid` command to get your personal chat ID or invite it into a group and use the `/getgroupid` command to get the group chat ID. Group IDs start with a hyphen, supergroup IDs start with `-100`.  
     Alternatively, you can get the chat ID directly from the bot API. Send *your* bot a command in the chat you want to use, then check `https://api.telegram.org/bot{YourBotToken}/getUpdates`, eg.  `https://api.telegram.org/bot111122223:7OpFlFFRzRBbrUUmIjj5HF9Ox2pYJZy5/getUpdates`
-
+4.  Set the bot token and the chat ID of the recipient in /etc/netdata/health_alarm_notify.conf (to edit it on your system run /etc/netdata/edit-config health_alarm_notify.conf), like this:
```

Comment:
Hello @kdvlr ,

Please, can you add add ``  around the file to call reader attention for the filename?

Best regards!

Reply:
For example, `/etc/netdata/health_alarm_notify.conf`

Reply:
Thanks. Done! Please review.